docs/tutorial/4-template.md

```diff
@@ -332,30 +334,31 @@ call.
 >>>     float: true,
 >>>     scope: "parent",
 >>>     clearance: 2em,
->>>   )[
->>>     #text(
->>>       17pt,
->>>       weight: "bold",
->>>       title,
->>>     )
+>>>     {
```

Comment:
The double indent is a bit annoying but I agree that it's more idiomatic in principle ...

Reply:
No, it's okay.